playbooks/rfcs.md

```diff
@@ -65,14 +83,25 @@ Create an issue, and work from this template:
 
     You can see our discussion [in slack here](/link/to/slack.com)
 
-Give a week.
+    ## How this RFC is implemented
```

Comment:
Not especially related to this PR, but maybe this doc should just link to [the issue template](https://github.com/artsy/README/blob/master/.github/ISSUE_TEMPLATE/rfc.md) rather than replicating it. That template should also reflect your updates.